packages/story-editor/src/components/checklist/karma/checklist.karma.js

```diff
@@ -612,12 +612,6 @@ describe('Checklist integration - Card visibility', () => {
         checkIfCardDoesNotExist
       );
 
-      // add poster image to see new problems
-      await addPosterImageWithIssues();
-      posterIssuesRequiringMediaUploadPermissions.forEach(
-        checkIfCardDoesNotExist
-      );
-
```

Comment:
As per https://github.com/google/web-stories-wp/pull/8697#issuecomment-901934149, this won't work.

If you remove this, the test does not suppose what it's to do.

What the test ought to do:

* Add a poster image, verify that no checklist check is added for contributors

What it now does:

* Verify that no checklist check is added (which of course it never will, because there is no poster image)

I provided some suggestion on how it might be solved in the comment.

If that doesn't work, perhaps sync with Prometheus pod to get advice.

Reply:
> If that doesn't work, perhaps sync with Prometheus pod to get advice.

None of the solutions did work. I do see the problem, the test is no longer valid, as can't set poster image like this. Not sure how did test ever worked.

Reply:
> Not sure how did test ever worked.

It worked because on `main` the edit button is visible and can be used to mock adding a poster image:

![Screenshot 2021-08-23 at 22 18 02](https://user-images.githubusercontent.com/841956/130513636-cc5d2866-2880-41bf-b526-69d365625024.png)

Please post a message on Slack, I'm sure Pea pod folks can help advise on this test.

We should either fix the test or remove it.

Reply:
I removed the test, because I believe the test was invalid in the first place. In the editor, you can not set poster image, if you can upload media. But this test allows you set the poster image anyone. This would not work in the real editor. I think these lines should be removed, as they were never validly mocked.

Reply:
> I removed the test [..]

You haven't removed the entire test though? If we can't find a solution, then the _entire test_ would need to be removed, as it's otherwise useless.

> [...]  because I believe the test was invalid in the first place. In the editor, you can not set poster image, if you can upload media. But this test allows you set the poster image anyone.

Yeah it was cheating a bit to set a poster image. Now we need to find a better way to mock it.

Let's wait for the feedback from the team on that one.

Do you recall what part of the example in https://github.com/google/web-stories-wp/pull/8697#issuecomment-901934149 did not work? Mocking the API provider like that in the fixture _should_ work?

Reply:
I played with it and I altered the snippet a little. The following will make the test run but we'd need to set the poster image on one of these properties. I thought it was `publisher_logo_url` but it's still showing blank for me 🤔 
```js
fixture = new Fixture({
        mocks: {
          getStoryById: () =>
            Promise.resolve({
              title: { raw: '' },
              status: 'draft',
              author: 1,
              slug: '',
              date: '2020-05-06T22:32:37',
              date_gmt: '2020-05-06T22:32:37',
              modified: '2020-05-06T22:32:37',
              excerpt: { raw: '' },
              link: 'http://stories.local/?post_type=web-story&p=1',
              preview_link: 'http://stories.local/?post_type=web-story&p=1',
              story_data: {
                version: DATA_VERSION,
                pages: [],
              },
              featured_media: 0, // TODO TODO TODO
              featured_media_url: '', // TODO TODO TODO
              publisher_logo_url:
                'http://stories.local/wp-content/plugins/web-stories/assets/images/logo.png',
              permalink_template: 'http://stories3.local/stories/%pagename%/',
              style_presets: { textStyles: [], colors: [] },
              password: '',
              _embedded: { author: [{ id: 1, name: 'John Doe' }] },
              _links: {
                'wp:action-assign-author': {
                  href: 'http://stories.local/wp-json/web-stories/v1/web-story/1',
                },
                'wp:action-delete': {
                  href: 'http://stories.local/wp-json/web-stories/v1/web-story/1',
                },
                'wp:action-publish': {
                  href: 'http://stories.local/wp-json/web-stories/v1/web-story/1',
                },
                'wp:action-unfiltered-html': {
                  href: 'http://stories.local/wp-json/web-stories/v1/web-story/1',
                },
              },
            }),
        },
      })
```

Reply:
`publisher_logo_url` is the publisher logo, as the name suggests. `feautred_media_url` would be the poster image (hence I added the `// TODO TODO TODO` comments there)

Reply:
Glad to see that my suggestion from https://github.com/google/web-stories-wp/pull/8697#issuecomment-901934149 _did_ work as expected.